NSubstitute.DbConnection/NSubstitute.DbConnection/MockQuery.cs

```diff
@@ -71,6 +71,12 @@ public IMockQueryResultBuilder Returns<T>(Func<QueryInfo, IEnumerable<T>> result
 
         public void Affects(Func<QueryInfo, int> rowCountSelector) => RowCountSelector = rowCountSelector;
 
+        public void Throws<T>(T exception)
+            where T : Exception
```

Comment:
This can be an expression body, it just needs the generic type constraint on a new line I think:

```
        where T : Exception => ResultSelectors.Add((typeof(T), _ => throw exception));
```

Reply:
The parentheses around the `throw` statement need to stay since it's not a `return` statement. Also it doesn't work well with the format document feature of Visual Studio as per previous comment but it has been done.